opendc-model-odc/jpa/src/main/resources/jpa/schema.xml

```diff
@@ -191,6 +191,19 @@
                     <cascade-persist/>
                 </cascade>
             </many-to-many>
+            <!-- TODO: this can be improved by reusing the `task_dependencies` table -->
+            <many-to-many name="dependents" target-entity="Task">
+                <join-table name="task_dependents">
+                    <join-column name="second_task_id"/>
```

Comment:
If you swap these two join column names, I think we don't need a `task_dependents` table.

Reply:
I think I fixed this now.